environments/development/ubuntu/group_vars/ubuntu.yml

```diff
@@ -0,0 +1,238 @@
+---
+# copyright Utrecht University
+
+# Ansible configuration
+ansible_user: vagrant                             # Administrative user on instance for Ansible
+ansible_ssh_private_key_file: vagrant/ssh/vagrant # Path to private key file of administrative user
+
+# Yoda configuration
+instance: ubuntu                                   # Name of Yoda instance, as defined in hosts file
+yoda_version: development                          # Git branch, for example: development or release-1.5
+yoda_environment: development                      # Yoda environment: development, testing, acceptance or production
+yoda_portal_fqdn: portal.yoda.test                 # Yoda Portal fully qualified domain name (FQDN)
+yoda_davrods_fqdn: data.yoda.test                  # Yoda Davrods WebDAV fully qualified domain name (FQDN)
+yoda_davrods_anonymous_enabled: true
+yoda_davrods_anonymous_fqdn: public.data.yoda.test # Yoda Davrods anonymous WebDAV fully qualified domain name (FQDN)
+
+default_yoda_schema: default-1                     # Default Yoda metadata scheme: core-0 or default-1
+yoda_prefix: UU01                                  # Prefix for internal portion of persistent identifier
+update_schemas: 1                                  # Update already installed schemas, formelements and stylesheets: yes (1) or no (0)
+credential_files: PLACEHOLDER                      # Location of Yoda credentials files
+
+# Research module configuration
+enable_research: true                              # Enable research module
+
+# OpenSearch module configuration
+enable_open_search: true                           # Enable OpenSearch module
+opensearch_server: portal.yoda.test
```

Comment:
The opensearch_server variable needs to be changed to 'combined.yoda.test' to be able to create the index (since we don't have a separate portal server in this dev environment)